build.gradle

```diff
@@ -84,6 +84,7 @@ subprojects {
             // Dealing with error "Wrong number of received repositories in state 'open'" (http://bit.ly/2ybracm)
             numberOfRetries = 50
             delayBetweenRetriesInMillis = 3000
+            stagingProfileId = getVersionForBuild()
```

Comment:
I don't think this is safe enough. If one build fails, the next build for the same version will use the old staging repository, which will probably contain some artifacts already, blocking new artifacts from being uploaded (the server returns a "Conflict" state, IIRC).

Ideally, this would be something like:

`"jaeger-java-client-%s-%s", version, timestamp`

This way, it's easy to find in the UI (the part up to the version) and it's almost guaranteed to be unique (the timestamp part).

Reply:
I think it's a good idea to add the timestamp. Also, I would encourage people dropping open staging repos after failed release. We should add that in release.md.